source/mem/memory_test.go

```diff
@@ -0,0 +1,303 @@
+package mem
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/golang-migrate/migrate/v4/source"
```

Comment:
Call `Test()` from `github.com/golang-migrate/migrate/v4/source/testing`. See `source/stub/stub_test.go` for an example